src/elm/Main.elm

```diff
@@ -930,7 +938,12 @@ update msg model =
               }
             , Cmd.batch
                 [ getPipelineConfig model org repo ref lineFocus refresh
-                , Navigation.replaceUrl model.navigationKey <| Routes.routeToUrl <| Routes.Pipeline org repo ref Nothing lineFocus
+                , case buildNumber of
+                    Just b ->
+                        Navigation.replaceUrl model.navigationKey <| Routes.routeToUrl <| Routes.BuildPipeline org repo b ref Nothing lineFocus
+
+                    Nothing ->
+                        Navigation.replaceUrl model.navigationKey <| Routes.routeToUrl <| Routes.Pipeline org repo ref Nothing lineFocus
```

Comment:
we use buildNumber == Nothing to route to the correct pipeline page, now that there are two pages that receive the same api responses